.vscode/launch.json

```diff
@@ -28,7 +28,7 @@
                 "-n",
                 "c/helloWorld",
                 "-t",
-                "emu2"
```

Comment:
I think we should keep it consistent and use `sfdxdebug` everywhere in this file